helmfile/values/falco.yaml.gotmpl

```diff
@@ -30,6 +44,13 @@ nodeSelector: {{- toYaml .Values.falco.nodeSelector | nindent 2  }}
 affinity: {{- toYaml .Values.falco.affinity | nindent 2  }}
 tolerations: {{- toYaml .Values.falco.tolerations | nindent 2  }}
 
+falcoctl:
+  artifact:
+    install:
+      enabled: false
+    follow:
+      enabled: false
+
```

Comment:
Would you like to write an issue about enabling this? Otherwise I'll do it.

Reply:
I can do that!

Reply:
There is now an issue on enabling `falcoctl` here: https://github.com/elastisys/compliantkubernetes-apps/issues/1669